tasks/build/archives.js

```diff
@@ -13,13 +13,13 @@ module.exports = function createPackages(grunt) {
 
   let archives = async (platform) => {
     // kibana.tar.gz
-    await exec('tar', ['-zchf', platform.tarPath, platform.buildName]);
+    await exec('tar', ['-zchf', relative(buildPath, platform.tarPath), platform.buildName]);
 
     // kibana.zip
     if (/windows/.test(platform.name)) {
-      await exec('zip', ['-rq', '-ll', platform.zipPath, platform.buildName]);
```

Comment:
What zip executable are you using? AFAIK there is no zip executable bundled with windows. So when I try to run the build process in your PR, I get an ENOENT error.